docs/2.3/extensions/heading-permalinks.md

```diff
@@ -72,6 +74,14 @@ The value of this nested configuration option should be a `string` that you want
 
 This should be a `string` you want prepended to HTML IDs.  This prevents generating HTML ID attributes which might conflict with others in your stylesheet.  A dash separator (`-`) will be added between the prefix and the ID.  You can instead set this to an empty string (`''`) if you don't want a prefix.
 
+### `attach_heading` (since 2.3.6)
+
+If this value is `true`, the `id` attributes will be written to the <h> tag instead of the <a>.
+
+### `heading_class` (since 2.3.6)
+
+If attach_heading is `true`, this class is added to the <h> tag.
+
```

Comment:
This will have to be released in 2.4.0 since it's technically a new feature, so let's move all the documentation changes to `docs/2.4/`